src/appleseed/renderer/kernel/rendering/shadingresultframebuffer.cpp

```diff
@@ -131,7 +126,7 @@ void ShadingResultFrameBuffer::merge(
     const float* APPLESEED_RESTRICT source_ptr = source.pixel(source_x, source_y);
     float* APPLESEED_RESTRICT dest_ptr = pixel(dest_x, dest_y);
 
-    for (size_t i = 0; i < m_channel_count; ++i)
+    for (size_t i = 0, e = m_channel_count; i < e; ++i)
```

Comment:
This is to help the reader and especially the compiler: this construction ensures that the loop's upper bound (`m_channel_count`) doesn't change during iteration, so the compiler doesn't have to reload it at each iteration.